runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -584,6 +584,12 @@ private void hop(boolean previous)
 				}
 			}
 
+			//Avoid switching to near-max population worlds to prevent failure
+			if (world.getPlayers() >= 1950)
```

Comment:
This should probably not be hardcoded as such.
Make it a final int and add it to the top of the class:
```java
private static final int MAX_PLAYER_COUNT = 1950;
```

```suggestion
			if (world.getPlayers() >= MAX_PLAYER_COUNT)
```

Reply:
Amended in the latest commit. Thanks.